mlir/include/mlir/Dialect/Tosa/Utils/QuantUtils.h

```diff
@@ -28,7 +28,7 @@ namespace tosa {
 
 /// From a scale value, computes multiplier and shift values
 /// for 16 or 32-bit scale widths.
-void computeMultiplierAndShift(double scale, int32_t &multiplier,
+bool computeMultiplierAndShift(double scale, int32_t &multiplier,
```

Comment:
As far as I can tell the only usage of this function (inside llvm-project) is here: https://github.com/llvm/llvm-project/blob/main/mlir/test/lib/Dialect/Tosa/TosaTestPasses.cpp#L166. Should we update this usage to check the return value?

Reply:
> Should we update this usage to check the return value?
I think both would work. I prefer changing the function signature to return a boolean which is more clear to me.

Reply:
right, thanks! updated the usage in LLVM.